substrate/frame/revive/src/evm/api/rlp_codec.rs

```diff
@@ -362,6 +412,27 @@ impl Encodable for Transaction4844Unsigned {
 	}
 }
 
+//See https://eips.ethereum.org/EIPS/eip-7702
+impl Encodable for Transaction7702Signed {
```

Comment:
I confirm that this is working with real-ethereum block data, from PR: https://github.com/paritytech/polkadot-sdk/pull/9538